packages/web-features-figuring/src/baseline/support.ts

```diff
@@ -0,0 +1,89 @@
+import { release } from "os";
+import { Browser } from "../browser-compat-data/browser";
+import { Compat } from "../browser-compat-data/compat";
+import { Feature } from "../browser-compat-data/feature";
+import { Release } from "../browser-compat-data/release";
+import { Qualifications } from "../browser-compat-data/supportStatements";
+
+type Support = Map<Browser, Release | undefined>;
+
+/**
+ * Map browsers to the release that most-recently introduced support for the feature.
+ */
+export function support(feature: Feature, browsers: Browser[]): Support {
+  const support: Support = new Map();
+  for (const b of browsers) {
+    const releases = feature.supportedBy({ only: [b] });
+
+    const unqualifiedReleases = [];
+    const qualifiedReleases = [];
+    for (const { release, qualifications } of releases) {
+      if (qualifications === undefined) {
+        unqualifiedReleases.push(release);
+      } else {
+        qualifiedReleases.push({ release, qualifications });
+      }
+    }
```

Comment:
It's not _super_ obvious here, but now `supportedBy()` returns `{ release, qualifications? }[]`, where qualifications are things like prefixes and partials. If a feature is supported in two support statements, then you'll get two `{ release, qualifications? }`. So, for example, if there's data showing support with both a prefix and without, then you'll get both `{ release, qualifications }` and `{ release }`. Tidy!

(I've just rejected the releases that have qualifications. We could get more clever with this, if we like.)